tests/kola/afterburn/platform-id

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+## kola:
+##   platforms: aws gcp
```

Comment:
```suggestion
##   platforms: aws gce
```

We should probably eradicate all uses of gce from the mantle code base to reduce confusion here

Reply:
Nice catch! That sounds better to me.